game/scripts/vscripts/Sections/Section1.ts

```diff
@@ -0,0 +1,75 @@
+import * as tg from "../TutorialGraph/index"
+import * as tut from "../Tutorial/Core"
+import { getPlayerHero } from "../util"
+
+let graph: tg.TutorialStep | undefined = undefined
+let graphContext: tg.TutorialContext
+
+enum Section01EntityKeys {
+    CustomMudGolem = "npc_custom_mud_golem",
+    MudGolemTop = "mud_golem_top",
+    MudGolemBot = "mud_golem_bot"
+}
+
+const onStart = (complete: () => void) => {
+    print("Starting", "Section01");
+    CustomGameEventManager.Send_ServerToAllClients("section_started", { section: SectionName.Section01 })
+
+    const playerHero = getPlayerHero();
+    if (!playerHero) error("Could not find the player's hero.");
+
+    let mudGolemMeetPosition = playerHero.GetAbsOrigin().__add(Vector(300, 800, 0))
+
+    graph = tg.seq(
+        tg.setUnitMoveCapability(() => playerHero, UnitMoveCapability.NONE),
+        tg.setCameraTarget(() => playerHero),
+        tg.spawnAndMoveUnit(Section01EntityKeys.CustomMudGolem, 
+            playerHero.GetAbsOrigin().__add(Vector(0, 1500, 0)), 
```

Comment:
You can also write this as `playerHero.GetAbsOrigin() + Vector(0, 1500, 0) as Vector`